civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -4582,3 +4583,18 @@ function _getManagerContacts($contact_id) {
 
     return $assigned_manager_contact_ids;
 }
+
+/**
+ * Function to determine if given manager-contact relationship is inactive or active
+ *
+ * @param array $relation
+ * @return boolean
+ */
+function relationActiveAndCurrent($relation){
+  if($relation['is_active'] == 1
```

Comment:
You can drop the if here and just `return $relation['is_active'] == 1 && ($relation['end_date'] >= date('Y-m-d') || $relation['end_date'] == '')`